pkg/provider/terraform/instance/apply.go

```diff
@@ -249,9 +255,12 @@ func (p *plugin) handleFiles(fns tfFuncs) error {
 }
 
 // doTerraformStateList shells out to run `terraform state list` and parses the result
-func doTerraformStateList(dir string) (map[TResourceType]map[TResourceName]struct{}, error) {
+func (p *plugin) doTerraformStateList(dir string) (map[TResourceType]map[TResourceName]struct{}, error) {
```

Comment:
Since you added the receiver here, do you still need to pass in dir?  Can you use p.Dir instead?

Reply:
Same comment for doTerraformShow and doTerraformShowForInstance

Reply:
Thank @craigyam, good catch.